docs/sources/alerting/manage-notifications/template-notifications/create-notification-templates.md

```diff
@@ -46,9 +46,9 @@ To create a notification template that contains more than one template:
 
 3. Write each template in the Content field, including `{{ define "name-of-template" }}` and `{{ end }}` at the start and end of each template.
 
-{{< figure max-width="940px" src="/static/img/docs/alerting/unified/new-notification-template-email-9-4.png" caption="New notification template" >}}
+   {{< figure max-width="940px" src="/static/img/docs/alerting/unified/new-notification-template-email-9-4.png" caption="New notification template" >}}
```

Comment:
Do these indentation changes do anything visually? Not a blocker, just curious what this does

Reply:
It adds a small amount of indentation on the page, but not much, 10 pixels or so. I asked the docs team and it appears there is no other way to fix number continuation though 🙁

Reply:
It adds a small amount of indentation on the page. I asked the docs team and it appears there is no other way to fix number continuation unfortunately. Here is a screenshot that shows the indentation

<img width="785" alt="Screenshot 2023-02-22 at 17 35 17" src="https://user-images.githubusercontent.com/85952834/220709601-5942c274-4d71-4bae-85ba-20faffb2e75b.png">